iceoryx_posh/test/integrationtests/test_posh_mepoo.cpp

```diff
@@ -330,8 +332,8 @@ class Mepoo_IntegrationTest : public Test
 
     MePooConfig memconf;
 
-    iox::popo::SenderPort senderPort{nullptr};
-    iox::popo::ReceiverPort receiverPort{nullptr};
+    iox::popo::PublisherPortUser* publisherPort{nullptr};
+    iox::popo::SubscriberPortUser* subscriberPort{nullptr};
```

Comment:
please use a `std::unique_ptr`

Reply:
Back to optional. Problem was `publisherPort = iox::popo::PublisherPortUser()`, which was using copy on Win, which was deleted. First solved by pointers and ` publisherPort = new iox::`. Now back to `optional` and `emplace`.